v2/pipeline.proto

```diff
@@ -23,3 +24,21 @@ message Pipeline {
   // Workflows contains one or more pipeline workflows.
   repeated PipelineWorkflow Workflows = 2 [ (gogoproto.jsontag) = "workflows", (gogoproto.moretags) = "yaml: \"workflows\"" ];
 }
+
+//for fallback pipeline
+
+
+// Pipeline represents a named collection of fallbackpipeline spec details.
+// FallbackPipeline represents a named collection of fallbackpipeline spec details.
+message FallbackPipeline {
+  // Metadata contains the name, namespace, labels and annotations of the
+  ObjectMeta Metadata = 1 [ (gogoproto.jsontag) = "metadata,omitempty", (gogoproto.embed) = true, (gogoproto.nullable) = false ];
+
+  // FallbackpipelineList contains one or more pipeline list.
+  //FallbackPipelineListflow Pipelinelist = 2 [ (gogoproto.jsontag) = "pipelinelist", (gogoproto.moretags) = "yaml: \"pipelinelist\"" ];
+  repeated ResourceReference pipelist =2 [ (gogoproto.jsontag) = "pipelinelist", (gogoproto.moretags) = "yaml: \"pipelinelist\"" ];
```

Comment:
Is it `Pipelinelist` or `pipelist`? I believe it should be `PipelineList` with camel-case.